test/functional/apps/dashboard/url_field_formatter.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import expect from '@kbn/expect';
+import { WebElementWrapper } from 'test/functional/services/lib/web_element_wrapper';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const { common, dashboard, settings, timePicker, visChart } = getPageObjects([
+    'common',
+    'dashboard',
+    'settings',
+    'timePicker',
+    'visChart',
+  ]);
+  const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
+  const testSubjects = getService('testSubjects');
+  const browser = getService('browser');
+  const fieldName = 'clientip';
+
+  const clickFieldAndCheckUrl = async (fieldLink: WebElementWrapper) => {
+    const fieldValue = await fieldLink.getVisibleText();
+    await fieldLink.click();
+    const windowHandlers = await browser.getAllWindowHandles();
+    expect(windowHandlers.length).to.equal(2);
+    await browser.switchToWindow(windowHandlers[1]);
+    const currentUrl = await browser.getCurrentUrl();
+    const fieldUrl = common.getHostPort() + '/app/' + fieldValue;
+    expect(currentUrl).to.equal(fieldUrl);
+  };
+
+  describe('Changing field formatter to Url', () => {
+    before(async function () {
+      await esArchiver.load('dashboard/current/kibana');
+      await kibanaServer.uiSettings.replace({
+        defaultIndex: '0bf35f60-3dc9-11e8-8660-4d65aa086b3c',
+      });
+      await common.navigateToApp('settings');
+      await settings.clickKibanaIndexPatterns();
+      await settings.clickIndexPatternLogstash();
```

Comment:
Should this be a separate test - while you set a field value to URL? Probably just a nit on my side? I am fine with it being in before part.

Reply:
I put to `before` because it is required by both follow-up tests. We actually do not test anything here, but update field for the tests